extensions/paperless-ngx/src/models/preferences.model.ts

```diff
@@ -3,4 +3,6 @@ export interface Preferences {
   apiToken: string;
   gridMode: boolean;
   dateFormat: string;
+  showCorrespondentInSubtitle: boolean;
+  showDateInSubtitle: boolean;
 }
```

Comment:
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`

Reply:
Just wanted to match it the existing code

Reply:
Would be better to remove it completely, do you mind doing that?

Reply:
🫡 Done